streams/src/main/java/org/apache/kafka/streams/kstream/KTable.java

```diff
@@ -17,73 +17,72 @@
 
 package org.apache.kafka.streams.kstream;
 
+import org.apache.kafka.common.annotation.InterfaceStability;
 import org.apache.kafka.common.serialization.Serde;
 import org.apache.kafka.streams.KeyValue;
 
 /**
- * KTable is an abstraction of a change log stream from a primary-keyed table.
+ * KTable is an abstraction of a <i>changelog stream</i> from a primary-keyed table.
  *
  * @param <K> Type of primary keys
  * @param <V> Type of value changes
  */
+@InterfaceStability.Unstable
 public interface KTable<K, V> {
 
     /**
-     * Creates a new instance of KTable consists of all elements of this stream which satisfy a predicate
+     * Create a new instance of {@link KTable} consists of all elements of this stream which satisfy a predicate.
      *
-     * @param predicate the instance of Predicate
-     * @return the instance of KTable with only those elements that satisfy the predicate
+     * @param predicate     the instance of {@link Predicate}
      */
     KTable<K, V> filter(Predicate<K, V> predicate);
 
     /**
-     * Creates a new instance of KTable consists all elements of this stream which do not satisfy a predicate
+     * Create a new instance of {@link KTable} consists all elements of this stream which do not satisfy a predicate
      *
-     * @param predicate the instance of Predicate
-     * @return the instance of KTable with only those elements that do not satisfy the predicate
+     * @param predicate     the instance of {@link Predicate}
      */
     KTable<K, V> filterOut(Predicate<K, V> predicate);
 
     /**
-     * Creates a new instance of KTable by transforming each value in this stream into a different value in the new stream.
+     * Create a new instance of {@link KTable} by transforming each value in this stream into a different value in the new stream.
      *
-     * @param mapper the instance of ValueMapper
-     * @param <V1>   the value type of the new stream
-     * @return the instance of KTable
+     * @param mapper        the instance of {@link ValueMapper}
+     * @param <V1>          the value type of the new stream
      */
     <V1> KTable<K, V1> mapValues(ValueMapper<V, V1> mapper);
 
     /**
-     * Sends key-value to a topic, also creates a new instance of KTable from the topic.
-     * This is equivalent to calling to(topic) and table(topic).
+     * Materialize this stream to a topic, also creates a new instance of {@link KTable} from the topic
```

Comment:
Same comments to all `KTable` methods that I already listed for the methods of the same name under `KStream` above.